.travis.yml

```diff
@@ -50,7 +50,7 @@ before_install:
   - echo "always_populate_raw_post_data = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

Comment:
I did, and just did again, but the sudo line didn't go away (it is present on my other project as well). i've removed it by hand however.